src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+  </head>
+  <body>
+Hello, world!
```

Comment:
```suggestion
    Hello, world!
```

Reply:
[DEMO LINK](https://lizapolokhina.github.io/layout_hello-world/)
[TEST REPOTR LINK](https://lizapolokhina.github.io/layout_hello-world/report/html_report/)

Reply:
use 3 dots on your first description in this PR to edit it